.github/workflows/release.yaml

```diff
@@ -202,7 +204,7 @@ jobs:
         max_attempts: 6
         retry_wait_seconds: 30
         command: >
-          brew bump-formula-pr "Formula/${FORMULA_NAME}.rb"
+          brew bump-formula-pr "${FORMULA_NAME}"
```

Comment:
This should work fine. You could also use `"${FORMULA_REPO}/${FORMULA_NAME}"` to always ensure that you specify the right formula. On the off chance that a formula called `jrnl-beta` is added to Homebrew/core, this might no longer work as expected. I doubt that will happen, though, so probably not worth worrying about.